lib/plugin/css-variables-urlresolver.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+
+const less = require("../thirdparty/less");
+
+const CSSVariablesUrlResolverPlugin = module.exports = function(config) {
```

Comment:
Why do we need another plugin? Can't we just re-use what the `variable-collector` plugin provides?

Reply:
Because of running it at a later point of time! isPreEvalVisitor vs. isPreVisitor - otherwise we have to write ugly if conditions in the coding

Reply:
Can you give me some more details here? VariableCollector also uses `isPreVisitor: true`, so I would expect it to work almost the same as this plugin.

Reply:
The logic for the detection of the URLs is pretty simple and it need to run in the PreVisitor phase. The CSS Var Collector is running in the PreEvalVisitor phase  to not resolve the vars. I don't want to run the full enchilada of the code of the CSS Var Collector again in the PreVisitor phase as this is unnecessary and IMO the code of the URL Collector is pretty trivial.

Reply:
ahh - got your point - I am checking the url-collector right now

Reply:
There's no need to run anything again.
The variables are already collected and available: https://github.com/SAP/less-openui5/blob/a6fea7c1eefc958dde13f5ba5e3acfcf4fd8d338/lib/index.js#L237

It's a different `tree.toCSS` run, but both are based on the exact same input, so I wouldn't expect any differences in the values. It's the list we already use for the inline theming parameters, so their values are exactly what we expect. That's why I don't expect the need for another plugin.

Reply:
url-collector is not what I meant, and I think it doesn't really fit for this purpose.

Reply:
`variable-collector` seems to be sufficient!